source/includes/kubernetes/_k8_configmaps.md

```diff
@@ -129,6 +129,61 @@ Return value:
 | `taskId` <br/>_string_     | The id corresponding to the create config maps task. |
 | `taskStatus` <br/>_string_ | The status of the operation.                         |
 
+<!-------------------- REPLACE A CONFIG MAP -------------------->
+
+#### Replace a config map
+
+```shell
+curl -X PUT \
+  -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/a_service/an_environment/configmaps/game-demo/default"
```

Comment:
Is it always the case the `id` is suffixed with the `namespace`? *game-demo/default*

Reply:
Also should we include the namespace information in the metadata of the example below?

Reply:
Yes, the id is created as such: `Strings.format("%s/%s", name, namespace);`
Namespace is an optional value so I didn't include it. Would it be more clear if I included it?

Reply:
Yeah I figured it must have been Optional. I was wondering maybe it will good if we include if like how you had done for the `_k8_secrets.md ` examples.

Reply:
Done! 234217fdcaaa20613f9e3dfa94fa3799077a728e

Reply:
Merci 🙌